src/styles/blocks/month.scss

```
@@ -0,0 +1,65 @@
+.month {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  gap: $gaps;
+
+  width: monthSize($day-size);
+
+  padding: 30px;
+
+  &__day {
+    box-sizing: border-box;
+    position: relative;
+    @include square($day-size);
+
+    background-color: #eee;
+    border: 1px solid black;
```

don't remember to create variables for colors

```suggestion
    background-color: $c-color;
    border: 1px solid $c-black;
```